andela/travel_tool_front
# #160749054 Fix Request Modal Background

#### What does this PR do?
fix modal request blank background

#### Description of Task to be completed?
- Fix request modal background
- Fix notification view details modal

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login into the application
- Click on any request on request table or approval table if there is any existing request.
- Check the `URL` to check if the URL contains `/requestId` when any requestId is clicked

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160749054](https://www.pivotaltracker.com/story/show/160749054)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-25 at 12 00 16 pm" src="https://user-images.githubusercontent.com/28366272/46010470-b224b400-c0ba-11e8-89aa-80f2f86e10c7.png">
<img width="1440" alt="screen shot 2018-09-25 at 12 00 33 pm" src="https://user-images.githubusercontent.com/28366272/46010472-b224b400-c0ba-11e8-87f5-48872bbffe6d.png">


#### Questions:
